docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: '3.6'
+
+services:
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
Let's run it in a similar way to what `make server-dev` does - with `
	DJANGO_SETTINGS_MODULE="pycon.dev_settings"` and `DEBUG=True`